CHANGES.md

```diff
@@ -6,6 +6,16 @@ list](https://github.com/trinodb/trino-python-client/tags), the
 [README](https://github.com/trinodb/trino-python-client/blob/master/README.md)
 and the [PyPI page](https://pypi.org/project/trino/).
 
+## Release 0.325.0
+
+* Queries with parameters now uses `EXECUTE IMMEDIATE` instead of `PREPARE`,
+  `EXECUTE` and `DEALLOCATE` if the server supports `EXECUTE IMMEDIATE`. If you
```

Comment:
> if the server supports `EXECUTE IMMEDIATE`

Do you mean python client automaticaly falls back to PREPARE when the server doesn't support EXECUTE IMMEDIATE statement?

Reply:
Yes.